web/styles/app_components.css

```diff
@@ -973,7 +973,10 @@ div.overlay {
         }
 
         .overlay_message_controls {
-            display: inline-block;
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            gap: 10px;
```

Comment:
Can you confirm that this doesn't make the "scheduled messages" UI that shares these classes look busted?

Reply:
Yes, they don't break the "scheduled messages" UI. 

The change makes the controls use a flexbox to make it easy to do modifications such as adding an icon; which was done in this PR. 

<details>

<summary> Screenshot </summary>

![image](https://github.com/zulip/zulip/assets/35286603/1074b7da-dd8d-4f28-a85f-38881f963df2)


</details>

Reply:
Great!